perception/map_based_prediction/README.md

```diff
@@ -151,23 +151,24 @@ If the target object is inside the road or crosswalk, this module outputs one or
-| `history_time_length`                       | double | Time span of object information used for prediction [s]                                                      |
-| `dist_ratio_threshold_to_left_bound`        | double | Conditions for using lane change detection of objects. Distance to the left bound of lanelet.                |
-| `dist_ratio_threshold_to_right_bound`       | double | Conditions for using lane change detection of objects. Distance to the right bound of lanelet.               |
-| `diff_dist_threshold_to_left_bound`         | double | Conditions for using lane change detection of objects. Differential value of horizontal position of objects. |
-| `diff_dist_threshold_to_right_bound`        | double | Conditions for using lane change detection of objects. Differential value of horizontal position of objects. |
+| Parameter                                   | Unit  | Type   | Description                                                                                                  |
+| ------------------------------------------- | ----- | ------ | ------------------------------------------------------------------------------------------------------------ |
+| `enable_delay_compensation`                 | [-]   | bool   | flag to enable the time delay compensation for the position of the object                                    |
+| `prediction_time_horizon`                   | [s]   | double | predict time duration for predicted path                                                                     |
+| `prediction_sampling_delta_time`            | [s]   | double | sampling time for points in predicted path                                                                   |
+| `min_velocity_for_map_based_prediction`     | [m/s] | double | apply map-based prediction to the objects with higher velocity than this value                               |
+| `min_crosswalk_user_velocity`               | [m/s] | double | minimum velocity use in path prediction for crosswalk users                                                  |
+| `dist_threshold_for_searching_lanelet`      | [m]   | double | The threshold of the angle used when searching for the lane to which the object belongs                      |
+| `dist_threshold_for_searching_lanelet`      | [m]   | double | The threshold of the distance used when searching for the lane to which the object belongs                   |
+| `delta_yaw_threshold_for_searching_lanelet` | [rad] | double | The threshold of the angle used when searching for the lane to which the object belongs                      |
+| `sigma_lateral_offset`                      | [m]   | double | Standard deviation for lateral position of objects                                                           |
+| `sigma_yaw_angle`                           | [rad] | double | Standard deviation yaw angle of objects                                                                      |
```

Comment:
According to `map_based_prediciton.param.yaml`, there is no parameter called `sigma_yaw_angle`, and it would be `sigma_yaw_angle_deg` in degree.

Reply:
@ktro2828 
Thank you for checking!
The explanatin doesn't need to be changed?
what I changed in this PR is swapping explanation for `dist_threshold_for_searching_lanelet` and `delta_yaw_threshold_for_searching_lanelet`.